### PR TITLE
Add GeneticChickengineering

### PR DESCRIPTION
**Pull request by kii-chan-reloaded**
## Description
This adds Genetic Chickengineering to the build server.

## Reason
Having automatic development builds will lighten the load on me to keep pushing releases for every point update. It was also suggested by @Seggan

### CODE REVIEW

**Comment by kii-chan-reloaded** [comment]
I couldn't figure out why all my builds were failing despite being valid JSON. Turns out a commit from a few days ago to `schema.json` had a missing quotation mark. May as well add that to this pull, I guess.

File: resources/schema.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
               "type": "boolean"
             },
             "createJar": {
-              "type": "boolean
+              "type": "boolean"
```
Comment (TheBusyBiscuit) [inline resources/schema.json:51]: Whooops thanks 😅

File: resources/repos.json [hunk 2]
```diff
@@ -1104,5 +1104,23 @@
...
+            "abandoned": false,
+            "createJar": true
+        },
+        "dependencies": {
+            "Minecraft Version(s)": {
+                "1": "1.16.x"
+            },
+            "Slimefun Version": {
+                "1": "<a class=\"link_info\" href=\"/builds/TheBusyBiscuit/Slimefun4/stable/#20\">RC-20</a>"
+            }
```
Comment (TheBusyBiscuit) [inline resources/repos.json]: Pretty sure your plugin requires Paper to be used, right?
If so, then you should add this as a dependency here.

File: resources/repos.json [hunk 3]
```diff
@@ -1121,6 +1121,9 @@
             "Slimefun Version": {
                 "1": "<a class=\"link_info\" href=\"/builds/TheBusyBiscuit/Slimefun4/stable/#20\">RC-20</a>"
             }
```
Comment (TheBusyBiscuit) [inline resources/repos.json]: Missing comma

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

**Review by TheBusyBiscuit — APPROVED** [review]

**Pull request merged**